src/testcafe/schema.json

```diff
@@ -162,6 +162,26 @@
         }
       }
     },
+    "compilerOptions": {
+      "type": "object",
+      "description": "[TestCafe] Compiler options",
+      "properties": {
+        "typescript": {
+          "type": "object",
+          "description": "[TestCafe] Typescript compiler options",
+          "properties": {
+            "customCompilerModulePath": {
+              "type": "string",
+              "description": "specify the path to the module that exports the compiler"
```

Comment:
Please copy the descriptions from the [related help topic](https://testcafe.io/documentation/402659/reference/testcafe-api/runner/compileroptions?search#header).